torch/optim/radam.py

```diff
@@ -379,22 +378,23 @@ def _multi_tensor_radam(
         del grouped_grads
 
         rect = [
-            _dispatch_sqrt(
-                (rho_t - 4)
-                * (rho_t - 2)
-                * rho_inf
-                / ((rho_inf - 4) * (rho_inf - 2) * rho_t)
-            )
-            if rho_t > 5
-            else 0
-            for rho_t in rho_t_list
+            torch.where(
```

Comment:
Hmm, not sure this is a good idea in the eager case? Anyway, it executes on CPU, but still might be a bit slow

Reply:
Putting this on the capturable path would help resolve this as well